integration-tests/src/test_loop/tests/in_memory_tries.rs

```diff
@@ -87,43 +85,7 @@ fn test_load_memtrie_after_empty_chunks() {
     }
     test_loop.run_instant();
 
-    let clients = node_datas
-        .iter()
-        .map(|data| &test_loop.data.get(&data.client_sender.actor_handle()).client)
-        .collect_vec();
-    let mut balances = accounts
-        .iter()
-        .cloned()
-        .map(|account| (account, initial_balance))
-        .collect::<HashMap<_, _>>();
-
-    let anchor_hash = *clients[0].chain.get_block_by_height(10002).unwrap().hash();
-    for i in 0..accounts.len() {
-        let amount = ONE_NEAR * (i as u128 + 1);
-        let tx = SignedTransaction::send_money(
-            1,
-            accounts[i].clone(),
-            accounts[(i + 1) % accounts.len()].clone(),
-            &create_user_test_signer(&accounts[i]).into(),
-            amount,
-            anchor_hash,
-        );
-        *balances.get_mut(&accounts[i]).unwrap() -= amount;
-        *balances.get_mut(&accounts[(i + 1) % accounts.len()]).unwrap() += amount;
-        let future = node_datas[i % num_clients]
-            .client_sender
-            .clone()
-            .with_delay(Duration::milliseconds(300 * i as i64))
-            .send_async(ProcessTxRequest {
-                transaction: tx,
-                is_forwarded: false,
-                check_only: false,
-            });
-        drop(future);
-    }
-
-    // Give plenty of time for these transactions to complete.
-    test_loop.run_for(Duration::seconds(40));
+    crate::test_loop::utils::execute_money_transfers(&mut test_loop, &node_datas, &accounts);
```

Comment:
nit: How about importing it? I don't think there is any risk of ambiguity given it takes test loop as the first param.